lib/expublish/changelog.ex

```diff
@@ -67,6 +62,34 @@ defmodule Expublish.Changelog do
     version
   end
 
+  @doc """
+  Build changelog entry title with version and ISO 8601 date.
+
+  Formats current or given `NaiveDateTime` to ISO 8601 date string.
+  Can be changed to date-time representation with the `--changelog-date-time` option.
+  """
+  @spec build_title(Version.t(), Options.t(), nil | NaiveDateTime.t()) :: String.t()
+  def build_title(version, options \\ %Options{}, date_time \\ nil)
+
+  def build_title(version, options, nil) do
+    build_title(version, options, NaiveDateTime.utc_now())
+  end
+
+  def build_title(version, %Options{changelog_date_time: true}, date_time) do
+    date_time
+    |> NaiveDateTime.truncate(:second)
+    |> NaiveDateTime.to_iso8601()
+    |> String.replace("T", " ")
+    |> format_title(version)
+  end
+
+  def build_title(version, _options, date_time) do
+    date_time
+    |> NaiveDateTime.to_date()
```

Comment:
You can skip this, Date.to_iso8601 takes anything that has year, month, day including NaiveDateTime and DateTime :)

Reply:
Makes total sense, should have at least tried if not checked :) thanks!